### ISSUE TITLE
Change "Apply to voice" labels to "Voice assignment"

### ISSUE THREAD

**Issue by avvvvve** [musescore/MuseScore#23537]
We're renaming what is currently called "Apply to voice" in master to the noun form, "Voice assignment". This will help us reference the property more easily in future features that utilize voice assignment for playback.

### We need to update the strings in these two locations:
- Properties for **dynamics** and **hairpins**
     Change "Apply to voice" to "Voice assignment"
     ![va_properties](https://github.com/musescore/MuseScore/assets/20806406/0a682ac3-c5b2-4c4b-8b5f-7d2098462b3f)

- Preferences
     In the 'Note input' section, change "Dynamics & hairpins" to "Voice assignment"
     ![va_prefs](https://github.com/musescore/MuseScore/assets/20806406/0cd630de-335e-46c8-af6b-1930fc9398a7)

**Comment by avvvvve** [comment]
@cbjeukendrup Assigned this to you since it's a small copy change and you've mentioned pushing strings to Transifex soon. Would be good to get this in for translation :)

**Comment by cbjeukendrup** [comment]
@avvvvve The only thing I noticed is that we now have a "voice" property and a "voice assignment" property. "Voice" means voice 1, 2, 3, or 4, and "voice assignment" means current instrument / current staff / current voice. 

I don't think it's really a problem because in the UI they are currently not presented in ways that could cause confusion between them, but perhaps it is something to keep in mind.

**Comment by avvvvve** [comment]
Yeah, that was definitely considered when we set out on designing this. I think there's enough of a connection between our pre-existing voice framework (i.e. the voice buttons in the toolbar and voice-specific colors) and the new property that they don't really need to be conceived as two separate features. For example, when a dynamic or hairpin is assigned to an individual voice, the corresponding button in the note input toolbar lights up. Of course the "All" options are new, but those won't apply to other element types.

Also, the pre-existing "voice" property was not actually controllable anywhere in the properties panel as far as I know.

While writing this I realized that the note input toolbar voice buttons are non-functional on dynamics/hairpins, so I logged an issue to allow that too https://github.com/musescore/MuseScore/issues/23543
